README.md

```diff
@@ -124,6 +124,22 @@ Using this template you can switch between the two using the Visual Studio "Targ
 When publishing your plugin you should build in Release mode for both x86 and x64 and publish both resulting dll's (e.g. `bin/Release/myPlugin.dll` and `/bin/Release-x64/MyPlugin.dll`)
 
 
+
+## Requirements
+
+  * works with .NET Runtime 4.0 and above 
+  * UNICODE plugins only.
+
+### v0.94.00
+The last version to work with Notepad++ v4.6.3 or below (newer versions of notepad++ use a different plugin structure, so you need to use a newer version of this framework)
```

Comment:
That should prob. be "v7.5.9 or below"
Perhaps we should mention that "For Notepad++ 7.6 to 7.6.2 no release works out of the box due to plugin directories changing with every release" and that the current code only works on 7.6.3 and above

Reply:
done